Source/WebCore/html/track/AudioTrack.idl

```diff
@@ -32,9 +32,9 @@
     Exposed=Window
 ] interface AudioTrack {
     readonly attribute DOMString id;
-    [EnabledConditionallyReadWriteBySetting=MediaSourceEnabled] attribute [AtomString] DOMString kind;
+    [EnabledConditionallyReadWriteBySetting=MediaSourceEnabled] readonly attribute [AtomString] DOMString kind;
```

Comment:
EnabledConditionallyReadWriteBySetting seems to have been added specifically for these attributes. It does seem like it was added by mistake as there's no need for these to be publicly writable, but then we should completely remove it here.

Reply:
Done and removed.